packages/rum-core/src/domain/rumEventsCollection/view/trackViews.spec.ts

```diff
@@ -190,6 +190,22 @@ describe('initial view', () => {
       expect(getViewUpdate(2).timings).toEqual({})
     })
 
+    it('should not update timings after session expires', () => {
```

Comment:
**❓ question:** ‏was there an issue with updating the initial view after session expiration?

Reply:
There was no issue, but also it was not updated after session expiration since the update got discarded in assembly. I chose preserve this behavior, but it's a bit arbitrary.

Updating the initial view after session expiration should be fine: it's an edge case so any potential negative impact would be quite limited, and it would make the code slightly simpler.

Reply:
it seems that we introduced that after an unexpected bump in session duration https://github.com/DataDog/browser-sdk/pull/1228 🤔
However, we were still tracking events count indefinitely at the time so that was maybe more related to event counts than to the initial view timings.
Maybe we could have something similar to `scheduleStopEventCountsTracking` and schedule the end of the timing updates when the initial view ends instead of doing that on session expiration.
wdyt?